linux/power_operation_scripts/power_operation_scripts.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   This test case is used for checking soft power operations
+#   When VMware tools is not installed or not running in VM, this test case result is 'Blocked'.
+#
+- name: power_operation_scripts
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: "Test case block"
+      block:
+        - name: "Test setup"
+          include_tasks: ../setup/test_setup.yml
+          vars:
+            skip_test_no_vmtools: true
+
+        - name: "Enable debug logging for VMware Tools"
+          include_tasks: ../utils/enable_vmtools_logging.yml
+
+        - name: "Set fact of customized script file"
```

Comment:
The task name is incorrect. It should be `Set fact of power script opertions`.  And vars names should be consistent. For example, you have used `power_script_op` as a kind of power operation script, here can use `power_script_ops` as a list of `power_script_op`. If you use `power_cmd_ops` as the list, then you'd better to use `power_cmd_op` to represent one kind of power operation script in set_power_script.yml.